### DISCUSSION TITLE
How to add a custom menu and other items on the LCD screen

### DISCUSSION THREAD

**Discussion by Jonny11bonk** [Jyers/Marlin#491] [Q&A] [answered]
I would love to see a tutorial how to add a custom image, text, menu screen and other options. I would like to add a brightness option only. What about custom languages?

**Comment by alphatrianguli** [comment, marked as answer]
The newest version of Jyers FW 1.3.0. has the LCD brightness option built-in.

**Reply by gokhanselen** [reply]
Hello do you have new menu icons

**Reply by alphatrianguli** [reply]
No

**Reply by Jonny11bonk** [reply]
> The newest version of Jyers FW 1.3.0. has the LCD brightness option built-in.

Yeah But I didn't want any other additions. Only the brightness function.

**Reply by Jyers** [reply]
@Mineplayerminer, so are you saying you're trying to add the brightness function to stock firmware?

**Reply by Jonny11bonk** [reply]
> @Mineplayerminer, so are you saying you're trying to add the brightness function to stock firmware?

Yes.

**Reply by Jyers** [reply]
You're welcome to try, but the stock creality code is very hard to work with. You'd be far better of using mine and adding anything else you want on top of it.
